de.manuel_kehl.go-for-it.yml

```diff
@@ -14,8 +14,6 @@ finish-args:
   # Access user files
   - --filesystem=home
 
-  # Ayatana indicator plugin
-  - --talk-name=org.kde.*
```

Comment:
To pass finish-args-wildcard-kde-talk-name linter

Reply:
> Exceptions allowed: Yes
> The manifest has a talk-name that is org.kde.*.
> Wildcard access to bus names in well known namespace is a security issue.

https://docs.flathub.org/docs/for-app-authors/linter/#finish-args-wildcard-kde-talk-name

Reply:
If this is needed, you have to ask for an exception. You can try running it with `flatpak run --log-session-bus` and check which org.kde name it wants.

Reply:
```
flatpak run --log-session-bus de.manuel_kehl.go-for-it 
C-65532: -> org.freedesktop.DBus fake wildcarded AddMatch for org.kde
```

I'm not sure.

Reply:
can you post the entire thing? If there is a setting to enable tray etc. in the app you will need to activate it.

Reply:
```
flatpak run --log-session-bus de.manuel_kehl.go-for-it
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake AddMatch for com.canonical.Unity.LauncherEntry
C-65535: -> org.freedesktop.DBus fake GetNameOwner for com.canonical.Unity.LauncherEntry
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65533: -> org.freedesktop.DBus fake wildcarded AddMatch for de.manuel_kehl.go-for-it
C-65532: -> org.freedesktop.DBus fake wildcarded AddMatch for org.kde
C-65531: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.de.manuel_kehl.go-for-it
C-65530: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return error org.freedesktop.DBus.Error.NameHasNoOwner from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65527: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65526: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65527
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65526
*SKIPPED*
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameOwnerChanged at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
C5: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C6: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C7: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C3
B-1: <- org.freedesktop.DBus return from C4
B-1: <- org.freedesktop.DBus return from C5
B-1: <- org.freedesktop.DBus return from C6
B-1: <- org.freedesktop.DBus return from C7
C8: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C8
C9: -> :1.78 call org.freedesktop.DBus.Properties.GetAll at /org/freedesktop/portal/desktop
B926: <- :1.78 return from C9
C10: -> :1.78 call org.freedesktop.portal.Settings.ReadAll at /org/freedesktop/portal/desktop
B930: <- :1.78 return from C10
C11: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C12: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
Filtering message due to arg0 org.gtk.vfs.Daemon, policy: 0 (required 2)
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C11
B-1: <- org.freedesktop.DBus return from C12
*REWRITTEN*
C13: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
Filtering message due to arg0 org.gtk.vfs.Daemon, policy: 0 (required 1)
B-1: <- org.freedesktop.DBus return from C13
*REWRITTEN*
C14: -> org.gtk.vfs.Daemon call org.gtk.vfs.MountTracker.ListMountableInfo at /org/gtk/vfs/mounttracker
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C14
*REWRITTEN*
C15: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C15
C16: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C17: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C18: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C16
B-1: <- org.freedesktop.DBus return from C17
B-1: <- org.freedesktop.DBus return from C18
C19: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C19
C20: -> :1.78 call org.freedesktop.DBus.Properties.GetAll at /org/freedesktop/portal/desktop
B931: <- :1.78 return from C20
C21: -> all signal org.gtk.Actions.Changed at /de/manuel_kehl/go_for_it
B9118: <- :1.23 call org.gtk.Actions.DescribeAll at /de/manuel_kehl/go_for_it
B9122: <- :1.23 call org.freedesktop.DBus.Properties.GetAll at /de/manuel_kehl/go_for_it
C22: -> :1.23 return from B9118
C23: -> :1.23 return from B9122
B9131: <- :1.23 call org.gtk.Actions.DescribeAll at /de/manuel_kehl/go_for_it/window/1
C24: -> :1.23 return from B9131
C25: -> org.freedesktop.portal.Desktop call org.freedesktop.portal.Notification.RemoveNotification at /org/freedesktop/portal/desktop
C26: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
C27: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
C28: -> org.freedesktop.DBus call org.freedesktop.DBus.ReleaseName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C26
B-1: <- org.freedesktop.DBus return from C27
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameLost at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameOwnerChanged at /org/freedesktop/DBus

```

Reply:
Doesn't show up in the log. Do you see the tray icon etc.? 

Might be a good idea to remove `org.kde.*` using Flatseal and then capture the log.

Reply:
Relevant?

```
Filtering message due to arg0 org.kde.StatusNotifierWatcher, policy: 0 (required 2)
*HIDDEN* (ping)
Filtering message due to arg0 org.kde.StatusNotifierWatcher, policy: 0 (required 1)
C26: -> all signal org.kde.StatusNotifierItem.NewIcon at /org/ayatana/NotificationItem/de_manuel_kehl_go_for_it
```

Reply:
Full log

```
flatpak run --log-session-bus de.manuel_kehl.go-for-it
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake AddMatch for com.canonical.Unity.LauncherEntry
C-65535: -> org.freedesktop.DBus fake GetNameOwner for com.canonical.Unity.LauncherEntry
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65533: -> org.freedesktop.DBus fake wildcarded AddMatch for de.manuel_kehl.go-for-it
C-65532: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.de.manuel_kehl.go-for-it
C-65531: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return error org.freedesktop.DBus.Error.NameHasNoOwner from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65527: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65527
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.RequestName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameOwnerChanged at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
C5: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C6: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C7: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C3
B-1: <- org.freedesktop.DBus return from C4
B-1: <- org.freedesktop.DBus return from C5
B-1: <- org.freedesktop.DBus return from C6
B-1: <- org.freedesktop.DBus return from C7
C8: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C8
C9: -> :1.77 call org.freedesktop.DBus.Properties.GetAll at /org/freedesktop/portal/desktop
B351: <- :1.77 return from C9
C10: -> :1.77 call org.freedesktop.portal.Settings.ReadAll at /org/freedesktop/portal/desktop
B355: <- :1.77 return from C10
C11: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C12: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
Filtering message due to arg0 org.gtk.vfs.Daemon, policy: 0 (required 2)
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C11
B-1: <- org.freedesktop.DBus return from C12
*REWRITTEN*
C13: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
Filtering message due to arg0 org.gtk.vfs.Daemon, policy: 0 (required 1)
B-1: <- org.freedesktop.DBus return from C13
*REWRITTEN*
C14: -> org.gtk.vfs.Daemon call org.gtk.vfs.MountTracker.ListMountableInfo at /org/gtk/vfs/mounttracker
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C14
*REWRITTEN*
C15: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C15
C16: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C17: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C18: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C16
B-1: <- org.freedesktop.DBus return from C17
B-1: <- org.freedesktop.DBus return from C18
C19: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C19
C20: -> :1.77 call org.freedesktop.DBus.Properties.GetAll at /org/freedesktop/portal/desktop
B356: <- :1.77 return from C20
C21: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C22: -> org.freedesktop.DBus call org.freedesktop.DBus.StartServiceByName at /org/freedesktop/DBus
Filtering message due to arg0 org.kde.StatusNotifierWatcher, policy: 0 (required 2)
*HIDDEN* (ping)
C23: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C24: -> all signal com.canonical.dbusmenu.LayoutUpdated at /org/ayatana/NotificationItem/de_manuel_kehl_go_for_it/Menu
B-1: <- org.freedesktop.DBus return from C21
B-1: <- org.freedesktop.DBus return from C22
*REWRITTEN*
B-1: <- org.freedesktop.DBus return from C23
C25: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
Filtering message due to arg0 org.kde.StatusNotifierWatcher, policy: 0 (required 1)
C26: -> all signal org.kde.StatusNotifierItem.NewIcon at /org/ayatana/NotificationItem/de_manuel_kehl_go_for_it
B-1: <- org.freedesktop.DBus return from C25
*REWRITTEN*
C27: -> all signal com.canonical.dbusmenu.LayoutUpdated at /org/ayatana/NotificationItem/de_manuel_kehl_go_for_it/Menu
C28: -> all signal org.gtk.Actions.Changed at /de/manuel_kehl/go_for_it
B5330: <- :1.19 call org.gtk.Actions.DescribeAll at /de/manuel_kehl/go_for_it
B5334: <- :1.19 call org.freedesktop.DBus.Properties.GetAll at /de/manuel_kehl/go_for_it
C29: -> :1.19 return from B5330
C30: -> :1.19 return from B5334
B5343: <- :1.19 call org.gtk.Actions.DescribeAll at /de/manuel_kehl/go_for_it/window/1
C31: -> :1.19 return from B5343

(de.manuel_kehl.go-for-it:2): Gtk-CRITICAL **: 20:04:52.937: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
C32: -> org.freedesktop.portal.Desktop call org.freedesktop.portal.Notification.RemoveNotification at /org/freedesktop/portal/desktop
C33: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
C34: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C33
B-1: <- org.freedesktop.DBus return from C34
C35: -> org.freedesktop.DBus call org.freedesktop.DBus.ReleaseName at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameLost at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameOwnerChanged at /org/freedesktop/DBus

```

Reply:
Yea can you try `--talk-name=org.kde.StatusNotifierWatcher`? Make sure `org.kde.*` is blocked through Flatseal.

Reply:
I see a three dots some kind of icon belongs to the go-for-it application.

![ayatanavs](https://github.com/user-attachments/assets/300e5f8b-ea59-465b-b804-b640c650c0d5)